lib/shopify_api/rest/redirect.ex

```diff
@@ -0,0 +1,118 @@
+defmodule ShopifyAPI.REST.Redirect do
+  @moduledoc false
+  @moduledoc """
+  Shopify REST API Redirect resources
+  """
+
+  alias ShopifyAPI.AuthToken
+  alias ShopifyAPI.REST
+  alias ShopifyAPI.REST.Redirect
```

Comment:
This appears to be an unused alias

Reply:
This and the defstruct were utilized; however, their usage contradicted the implementation of every other module in the library & so were removed from the logic. These two vestiges will be removed as well so that the module keeps in spirit with the rest of the library.

Reply:
Thanks for understanding, we talked back and forth about including structs for Shopify objects but always come back to 1) How much do we really get 2) it is going to be quite time consuming to create them all.

Reply:
yeah, i totally understand that value vs overhead choice; Shopify's versioning allows a greater flexibility here & really at the end of it all, these structure enforcement patterns seem to end up having little ongoing value other than documenting the expected return structure. For a stronger typed language, these kinds of definitions are a must, but the value here is nominal. We have a pretty good mapping system provided by Elixir's JSON parsers & I understand the choice here.

Thanks for the opportunity to talk it through :)